plugins/modules/source_control/github/github_repo.py

```diff
@@ -42,14 +42,16 @@
   description:
     description:
     - Description for the repository.
-    - Defaults to empty when creating a new repository.
+    - Defaults to empty if force_defaults=true, which is the default in this module.
```

Comment:
```suggestion
    - Defaults to empty if I(force_defaults=true), which is the default in this module.
```
please apply to the other cases as well.